src/command_modules/azure-cli-resource/tests/test_locks.py

```diff
@@ -0,0 +1,39 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import unittest
+
+from azure.cli.testsdk import ScenarioTest, ResourceGroupPreparer
+
+
+class StorageAccountTests(ScenarioTest):
```

Comment:
Why is this class called StorageAccountTests??  Shouldn't it be "LockScenarioTests" or something similar?